torch/lib/c10d/test/ProcessGroupNCCLTest.cpp

```diff
@@ -9,6 +9,7 @@
 #include <c10/cuda/CUDAGuard.h>
```

Comment:
We probably need to add tests to ProcessGroupMPITest to validate the profiling works correctly for that as well.